examples/autoencoder.py

```diff
@@ -13,20 +13,22 @@
 parser = argparse.ArgumentParser()
 parser.add_argument('--model', type=str, default='GAE')
 parser.add_argument('--dataset', type=str, default='Cora')
+parser.add_argument('--epochs', type=int, default=400)
```

Comment:
By the way, I added an argument for the number of training iterations, instead of the previous hardcoded value.